doc/source/index.rst

```diff
@@ -32,13 +32,13 @@ which is no longer actively developed or supported.
 .. hlist::
    :columns: 2
 
-   - Backup and restore for both classic non-sharded replica sets and sharded clusters
+   - Logical-style backup and restore. Physical-style (a.k.a. 'hot') backup and restore not supported
```

Comment:
```suggestion
   - Logical backup and restore. Physical (a.k.a. 'hot') backup and restore not supported
```


I would not use "-style" here. It's not searchable by Google as compared to just "physical" and "logical backup"